core/src/autogluon/core/scheduler/hyperband.py

```diff
@@ -169,11 +206,22 @@ class HyperbandScheduler(FIFOScheduler):
         - 'rungs_and_last': Results at rung levels, plus the most recent result.
             This means that in between rung levels, only the most recent result
             is used by the searcher. This is in between
+    rung_system_per_bracket : bool
+        This concerns Hyperband with brackets > 1. When starting a job for a
+        new config, it is assigned a randomly sampled bracket. The larger the
+        bracket, the larger the grace period for the config. If
+        `rung_system_per_bracket` is True, we maintain separate rung level
+        systems for each bracket, so that configs only compete with others
+        started in the same bracket. If False, we use a single rung level
+        system, so that all configs compete with each other. In this case, the
+        bracket of a config only determines the initial grace period, i.e. the
+        first milestone at which it starts competing with others.
```

Comment:
consider adding sentence clarifying when we might prefer rung_system_per_bracket = True vs False